website/django/data/searchtest.py

```diff
@@ -479,22 +433,13 @@ def test_show_deleted_not_true(self):
 
         response_data = json.loads(response.content)
 
-        self.assertEqual(response_data['count'], 1)
+        #0 variants should be in response_data['count']
+        self.assertEqual(response_data['count'], 0)
```

Comment:
why isn't existing_variant returned? if the issue is that new_variant_deleted and existing_variant have the same genomic_coordinate, the genomic_coordinate for new_variant_deleted should be different or the test is confusing.